spec/requests/api/v1/sessions/create_spec.rb

```diff
@@ -1,6 +1,28 @@
 require 'rails_helper'
 
 describe 'POST api/v1/users/sign_in', type: :request do
+  shared_context 'log in successfully' do
```

Comment:
I think this should be `shared_examples`.

Reply:
Nice, I have been using shared contexts wrong for a long time, thanks!!!